extensions/lifecycle/tasks/LifecycleRetriggerRestoreTask.js

```diff
@@ -0,0 +1,40 @@
+'use strict'; // eslint-disable-line
+
+const { LifecycleRequeueTask } = require('./LifecycleRequeueTask');
+
+class LifecycleRetriggerRestoreTask extends LifecycleRequeueTask {
+    /**
+     * Process a lifecycle object entry
+     *
+     * @constructor
+     * @param {LifecycleObjectProcessor} proc - object processor instance
+     */
+     constructor(proc) {
+        super(proc);
+        this.processName = 'retriggerRestore';
```

Comment:
should be just 'restore'

Reply:
or maybe we should keep 'transition' as well here: @nicolas2bert what do you think?

Reply:
> (btw this is not a "process" as in Linux process, but the "workflow" for use in lifecycle metrics: other values are 'transition', 'archive', 'expiration'...)

I took this name from the function where we use this variable https://github.com/scality/backbeat/blob/b33d280015eecdf697ba4f292474236d4b44399b/extensions/lifecycle/LifecycleMetrics.js#L91
Didn't know about the other values though :)

Reply:
Lifecycle Retrigger Restore Task is a component of the "restore workflow".
Additionally, we need to incorporate "restore workflow" metrics into the Lifecycle Update Expiration Task.